group_vars/location_noki/networks.yml

```diff
@@ -0,0 +1,87 @@
+---
+ipv6_prefix: '2001:bf7:760:300::/56'
+
+# got following prefixes:
+# Router: 10.31.174.0/26
+# --DHCP: 10.31.174.0/27
+# --DHCP: 10.31.174.32/28
+# --MESH: 10.31.174.48/30
+# --UPLK: 10.31.174.52/30
+# --MGMT: 10.31.174.56/29
+
+# Disable noping
+dhcp_no_ping: false
+
+networks:
+  # DHCP with filtering and isolation
+  - vid: 40
+    role: dhcp
+    inbound_filtering: true
+    enforce_client_isolation: true
+    # untagged: true
+    prefix: 10.31.174.0/27
+    ipv6_subprefix: 0
+    assignments:
+      noki-core: 1
+
+  # DHCP without filtering and isolation
+  - vid: 50
+    role: dhcp
+    name: nokihost
+    prefix: 10.31.174.32/28
+    ipv6_subprefix: 50
+    assignments:
+      noki-core: 1
+
+  # MESH - 5 GHz 802.11s
+  - vid: 20
+    role: mesh
+    name: mesh_11s_5ghz
+    prefix: 10.31.174.48/32
+    ipv6_subprefix: -1
+    mesh_ap: noki-core
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+  # MESH - 2.4 GHz 802.11s
+  - vid: 21
+    role: mesh
+    name: mesh_11s_2ghz
+    prefix: 10.31.174.49/32
+    ipv6_subprefix: -2
+    # make mesh_metric(s) for 2GHz worse than 5GHz
+    mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.8']
+    mesh_ap: noki-core
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+  # MGMT
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.174.56/29
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      # 10.31.174.57/32
+      noki-core: 1
+
+  # UPLK - uplink via Wireguard and local internet connection
+  - vid: 43
```

Comment:
Die anderen Standorte haben 41 für das private DHCP und 50 für den Uplink

Reply:
> Die anderen Standorte haben 41 für das private DHCP und 50 für den Uplink

Es ist nicht so einheitlich, wie du das gerne hättest: https://github.com/freifunk-berlin/bbb-configs/search?p=2&q=%22role%3A+ext%22

Wenn du magst, dann hallte das als Konvention in der Dokumentation fest, dann können wir das vereinheitlichen.